src/components/course/data/tests/hooks.test.jsx

```diff
@@ -138,87 +146,9 @@ const createGlobalLocationMock = () => {
 };
 const mockPreventDefault = jest.fn();
 
-const mockAuthenticatedUser = authenticatedUserFactory();
+const mockAuthenticatedUser = { authenticatedUser: authenticatedUserFactory() };
```

Comment:
nit: This value is more representing the `AppContext` value, which happens to contain `authenticatedUser`. But the `mockAuthenticatedUser` itself likely should just be `authenticatedUserFactory()`.